File: BackEnd/Models/Bagagem.cs
using System.ComponentModel.DataAnnotations;

public class Bagagem
{
    [Key]
    public int Id { get; set; }
    public decimal Peso { get; set; }
    public Passageiro? Passageiro { get; set; }
    public int PassageiroId { get; set; }
}

File: FrontEnd/AppPassagem/src/app/models/Bagagem.ts
import { Passageiro } from "./Passageiro";

export class Bagagem {
    id: number = 0;
    peso: number = 0;
    passageiro: Passageiro;
    passageiroId: number = 0;

    constructor(peso: number, passageiro: Passageiro)
    {
        this.peso = peso
        this.passageiro = passageiro
    }
}